content/guides/trash/permanently-delete-a-file.md

```diff
@@ -0,0 +1,19 @@
+---
+rank: 2
+related_endpoints:
+ - delete_files_id
+related_guides: []
+required_guides: []
+related_resources:
+  - files
+---
+
+# Permanently delete a file
+
+Once a file has been moved to the trash, it will remain in the trash for 30
+days, by default, before being purged. Administrators of Business or
+Enterprise accounts can alter the purge window. If you wish to permanently
+delete the file from the trash before the purge window expires, make a `DELETE`
+request to `/files/:file_id/trash` using the trashed file's `ID`.
+
+<Samples id='delete_file_id_trash' />
```

Comment:
The sample ID maps exactly the the strings in the path, so `DELETE /files/:id/trash` becomes `delete_files_id_trash`. Note the plural of `files`.